src/components/Title/Title.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { forwardRef } from 'react';
+
+type TitleType = {
+  id: string;
+  classname?: string;
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onClick?: () => void;
+  readonly: boolean;
+};
+
+const Title = forwardRef(
```

Comment:
Do you need this component ? It ultimately is only and input, you do not do anything special/any sideefect here